src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+public class Ui {
```

Comment:
Perhaps you could encapsulate more of the Ui related details within the Ui class?